service/controller/v24/resource/ebsvolume/delete.go

```diff
@@ -15,23 +15,35 @@ import (
 // EnsureDeleted detaches and deletes the EBS volumes. We don't return
 // errors so deletion logic in following resources is executed.
 func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
-	customObject, err := key.ToCustomObject(obj)
+	cr, err := key.ToCustomObject(obj)
```

Comment:
<3

Reply:
Ah yeah, I derailed a little. More to come though. :trollface: